app/src/main/java/com/example/dynamicdiet/MainActivity.kt

```diff
@@ -47,12 +50,12 @@ class MainActivity : ComponentActivity() {
                                 //modifier = Modifier.fillMaxSize(),
                                 color = MaterialTheme.colors.background
                             ) {
-                                InputWeight()
+                                inputWeight()
```

Comment:
I really want to agree with you on this, as it is normally the case, but composable functions should be nouns, and PascalCase, believe it or not.  
https://developer.android.com/kotlin/style-guide